Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.py

```diff
@@ -807,15 +809,47 @@ def endpoint_isolation_remove_command(client: Client, args: Dict[str, Any]) -> C
     return command_results
 
 
-def fetch_incidents(client: Client) -> list:
+def fetch_alerts_related_incident(client: Client, incident_id: str) -> list[Any]:
+    """
+    Returns the alerts that are associated with an incident.
+    """
+    alerts = []
+    has_next = True
+    page_number = 0
+
+    while has_next:
+        try:
+            response_body = client.incident_list_alerts_request(
+                page_number=str(page_number),
+                id_=incident_id,
+                page_size=None
+            )
+            alerts.extend(response_body.get('items', []))
+            has_next = response_body.get('hasNext', False)
+            page_number += 1
+        except DemistoException as e:
+            raise DemistoException(f"Error occurred while fetching alerts: {str(e)}")
```

Comment:
this hides the traceback. instead, use `demisto.error` for the message (without the `str(e)` suffix), then `raise` (with no arguments). You can remove the `as e` in this case.